File: Code_samples/C#/Fhi.OpenApiClient/Models/DataRequest.cs
namespace Fhi.OpenApiClient.Models
{
    internal class DataRequest
    {
        public Dimension[]? Dimensions { get; set; }
        public ResponseClass? Response { get; set; }

        internal class Dimension
        {
            public string? Code { get; set; }

            public string? Filter { get; set; }

            public string[]? Values { get; set; }
        }

        internal class ResponseClass
        {
            public string? Format { get; set; }

            public int? MaxRowCount { get; set; }
        }
    }
}

File: Code_samples/Kotlin/FhiOpenApiClient/src/main/kotlin/models/DataRequest.kt
package models

import kotlinx.serialization.Serializable

@Serializable
internal class DataRequest {
    var dimensions: Array<Dimension>? = null
    var response: ResponseClass? = null

    @Serializable
    internal class Dimension {
        var code: String? = null
        var filter: String? = null
        var values: Array<String>? = null
    }

    @Serializable
    internal class ResponseClass {
        var format: String? = null
        var maxRowCount: Int? = null
    }
}